deployments/base-goerli/Base_SpokePool.json

```diff
@@ -0,0 +1,1620 @@
+{
```

Comment:
This wasn't generated for some reason during deployment (I think the proxies don't generate these files for some reason), so i just copied the optimism auto-generated file. No need to review this.